bitnami/redis-cluster/templates/tls-secret.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if (include "redis-cluster.createTlsSecret" .) }}
+{{- $ca := genCA "redis-cluster-ca" 365 }}
+{{- $releaseNamespace := .Release.Namespace }}
+{{- $clusterDomain := .Values.clusterDomain }}
+
+{{- $fullname := include "common.names.fullname" . }}
+{{- $serviceName := include "common.names.fullname" . }}
+{{- $headlessServiceName := printf "%s-headless" (include "common.names.fullname" .) }}
+{{- $altNames := list (printf "*.%s.%s.svc.%s" $serviceName $releaseNamespace $clusterDomain) (printf "%s.%s.svc.%s" $serviceName $releaseNamespace $clusterDomain) (printf "*.%s.%s.svc.%s" $headlessServiceName $releaseNamespace $clusterDomain) (printf "%s.%s.svc.%s" $headlessServiceName $releaseNamespace $clusterDomain) $fullname }}
+{{- $crt := genSignedCert $fullname nil $altNames 365 $ca }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "common.names.fullname" . }}-crt
```

Comment:
I added `$fullname` to be used in the `altNames` list.

I think using it to assign the secret name could be confusing and wouldn't really make much difference.

Reply:
I personally don't like to follow two different approaches for setting the full name in the same file, we are using here different calls to `common.names.fullname`, and also we use `$fullname` several times.
Since it seems we added the same snippet to all charts I will accept it anyways.